stylesheets/_githubStyle.scss

```diff
@@ -0,0 +1,105 @@
+/**
+ * Shades of Purple Theme — for Highlightjs.
```

Comment:
Hm.. copied the wrong stylesheet over and didn't change the name of this file to properly reflect it. Will change in a later commit

Reply:
`highlight.js` provides many themes, so I think the user have the options to choose one. At the very minimum, a dark and a light mode should be available.